app/assets/stylesheets/common/_base.scss

```diff
@@ -284,3 +284,13 @@ tr:nth-child(odd) {background: #FFF}
   background-color: $review-gray;
   color: white;
 }
+.button_container {
```

Comment:
These css are specific to certain views, these should go in their respective scss files rather than in the general one (_base.scss).  Also the css names should be more descriptive as to what they are styling.  (e.g., review_details_button) unless these style are being used in other places.

Reply:
Done!